aries-site/src/examples/templates/dashboards/components/Legend.js

```diff
@@ -0,0 +1,30 @@
+import { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, List, Text, ThemeContext } from 'grommet';
+
+export const Legend = ({ values, ...rest }) => {
+  const theme = useContext(ThemeContext);
+  const size = theme.global.edgeSize.small;
+
+  return (
+    <List
+      data={values}
+      defaultItemProps={{ pad: { vertical: 'xxsmall' } }}
+      {...rest}
+    >
+      {datum => (
+        <Box direction="row" gap="small" justify="between">
+          <Box direction="row" align="center" gap="xsmall">
+            <Box background={datum.color} height={size} width={size} round />
+            <Text>{datum.label}</Text>
```

Comment:
Designs call for the form value weight. We haven't always been applying that directly on text (weight={500} I think)--so would just be good to confirm if we want to do that or if designs should change to default text.

Reply:
Confirming.

Reply:
No confirmation yet. Propose we move forward as is and adjust in subsequent PR once we get resolution.

Reply:
agree we can change after